Library/Formula/fpc.rb

```diff
@@ -28,11 +28,15 @@ def install
 
     # Prevent non-executable audit warning
     rm_f Dir[bin/"*.rsj"]
+
+    # Generate a default fpc.cfg to set up unit search paths
+    system "#{bin}/fpcmkcfg", "-p", "-d", "basepath=#{lib}/fpc/#{version}", "-o", "#{prefix}/etc/fpc.cfg"
```

Comment:
You should use `#{etc}` here instead or generate it into `buildpath` and then use `etc.install`.